vllm_ascend/worker/model_runner.py

```diff
@@ -52,8 +52,6 @@
 from vllm.multimodal import (MULTIMODAL_REGISTRY, BatchedTensorInputs,
```

Comment:
For V0 code, this file can be removed totally. I think we can remove V0 first. Then remove the related `prompt adapter` code in V1